yt/visualization/plot_container.py

```diff
@@ -257,7 +257,10 @@ def __init__(self, data_source, figure_size, fontsize):
             self.figure_size = float(figure_size[0]), float(figure_size[1])
         else:
             self.figure_size = float(figure_size)
-        font_path = matplotlib.get_data_path() + "/fonts/ttf/STIXGeneral.ttf"
+
+        # MPL >= 3.4.3 returns a Path
+        font_path = os.path.join(matplotlib.get_data_path(), "fonts", "ttf", "STIXGeneral.ttf")
+
```

Comment:
Actually the comment turns out to be ill-informed, so I suggest cleaning it up.
```suggestion
        font_path = os.path.join(matplotlib.get_data_path(), "fonts", "ttf", "STIXGeneral.ttf")
```